src/app.ts

```diff
@@ -6,6 +6,7 @@ import { logger } from './logger';
 import { router } from './routes';
 
 const app = express();
+
```

Comment:
These empty lines here are set by the https://eslint.org/docs/latest/rules/padding-line-between-statements which I think helps a lot with readability.